packages/header-checker-lint/src/header-checker-lint.ts

```diff
@@ -74,6 +75,23 @@ class Configuration {
 
 export = (app: Application) => {
   app.on('pull_request', async context => {
+    let remoteConfiguration = DEFAULT_CONFIGURATION;
+    try {
+      const candidateConfiguration = await context.config<ConfigurationOptions>(
+        WELL_KNOWN_CONFIGURATION_FILE
```

Comment:
Do you think it would be worth pulling this logic into a helper that all bots share at some point, seems like we're starting to collect a few lines of code.

Perhaps a tracking ticket to do this?

Reply:
Absolutely, but maybe not in this pull request. It might even be nice to validate against a schema like SLO bot does.

Opened https://github.com/googleapis/repo-automation-bots/issues/870